### ISSUE TITLE
NFS kernel modules not included

### ISSUE THREAD

**Issue by bgamari** [parallella/parabuntu#12]
The kernel configuration appears not to set `CONFIG_NFS`, precluding use as an NFS client.

**Comment by olajep** [comment]
Short answer:
The parabuntu kernel config has:
```
CONFIG_NFS_FS=y
```

Longer answer: 
Two files are sourced from be build scripts:  
https://github.com/parallella/parallella-linux/blob/parallella-linux-2016.11/arch/arm/configs/parallella_defconfig 

https://github.com/parallella/parabuntu/blob/master/kernel.config

// Ola

**Comment by bgamari** [comment]
Hmm, interesting. I'm not sure what to make of this in that case,

```
ben@parallella:~$ sudo apt-get install nfs-common
Reading package lists... Done
Building dependency tree       
Reading state information... Done
nfs-common is already the newest version.
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
ben@parallella:~$ sudo mount -tnfs 192.168.3.1:/exports/data /mnt/server
mount.nfs: No such device
ben@parallella:~$ cat /etc/issue
Ubuntu 15.04 \n \l
```

**Comment by bgamari** [comment]
Strangely enough, `gunzip -c /proc/config.gz` has no sign of `CONFIG_NFS_FS` whatsoever.

**Comment by olajep** [comment]
You're right! Turns out that CONFIG_NETWORK_FILESYSTEMS must be explicitly selected. Otherwise the nfs options will be ignored.

Fixed here:
https://github.com/parallella/parallella-linux/commit/163953f2bfc19037420615962aa88314840f5a73

New uImage w/ nfs:
[uImage.zip](https://github.com/parallella/parabuntu/files/862367/uImage.zip)

Recipe:

1. Download uImage.zip
2. Unpack
3. scp uImage to parallella board
4. Copy to boot partition
```
parallella@parallella:~ $ sudo mount /dev/mmcblk0p1 /mnt
parallella@parallella:~ $ sudo cp uImage /mnt
parallella@parallella:~ $ sync
parallella@parallella:~ $ sudo umount /mnt
```
5. Reboot parallella

6. Install nfs-common:  
```
$ sudo apt-get install nfs-common
```

// Ola

**Comment by bgamari** [comment]
Thanks!

**Comment by bgamari** [comment]
Indeed this works; I'll allow you to close the ticket when this kernel goes into an official release.

**Comment by olajep** [comment]
Good to hear that,

Leave this open until the next Parabuntu release.
